### PR TITLE
[TEST](indice_convolution_forward):add api check cases for indice_convolution_forward

### PR DESCRIPTION
**Pull request by Shin-Wang**
…volution_forward

Thanks for your contribution and we appreciate it a lot. 

## 1. Motivation

Please describe your motivation and the goal you want to achieve through this pull request.

## 2. Modification

Please briefly describe what modification is made in this pull request, and indicate where to make the modification.

## 3. Test Report

If you want to know how to do operator testing, you can see [GTest-User-Guide-zh](https://github.com/Cambricon/mlu-ops/blob/master/docs/GTest-User-Guide-zh.md).

#### 3.1.1 Accuracy Acceptance Standard

For static threshold standard details, see: [MLU-OPS Accuracy Acceptance Standard](https://github.com/Cambricon/mlu-ops/blob/master/docs/MLU-OPS-Accuracy-Acceptance-Standard.md).

- [ ] diff1: diff1 <= 3e-3
- [ ] diff2: diff2 <= 3e-3

#### 3.1.2 Operator Scheme checklist

|     No.        |                 Details              |            Check Results             |
|----------------|--------------------------------------|--------------------------------------|
|        1       |Supported hardware                    |             MLU370<br>MLU590         |
|        2       |Job types                             |          block <br> U1 <br> U4       |
|        3       |Layouts                               |          NHWC 、NCHW、ARRAY etc      |
|        4       |Whether multi-dimensions are supported|                                      |
|        5       |Whether element zero is supported     |                                      |
|        6       |Data type(half/float)                 |           half / float etc           |
|        7       |Whether there is size limit           |                                      |

#### 3.1.3 New Feature Test

If you have checked the following items, please tick the relevant box.

- [ ] Data type test
- [ ] Multidimensional tensor test
- [ ] Layout test
- [ ] Different size/integer remainder end segment/alignment misalignment test
- [ ] Zero dimensional tensor test/zero element test
- [ ] stability test
- [ ] Multiple platform test
- [ ] Gen_case module test
- [ ] Nan/INF tests 
- [ ] Bug fix tests
- [ ] For memory leak check details, see[GTest-User-Guide-zh](https://github.com/Cambricon/mlu-ops/blob/master/docs/GTest-User-Guide-zh.md).
- [ ] For code coverage check details, see: [GTest-User-Guide-zh](https://github.com/Cambricon/mlu-ops/blob/master/docs/GTest-User-Guide-zh.md).
- [ ] For I/O calculation efficiency check details, see: [MLU-OPS Performance Acceptance Standard](https://github.com/Cambricon/mlu-ops/blob/master/docs/MLU-OPS-Performance-Acceptance-Standard.md).

#### 3.1.4 Parameter Check

When a new operator is submitted, the test points are given and the test results are stated.

|                   Test Point                    | Acceptance Standard | Test Result (Error Message) |
| ----------------------------------------------- | --------------------| --------------------------- |
| Whether it conforms to the operator restriction |     Normal error    |                             |
| Whether illegal parameters are passed           |     Normal error    |                             |

### 3.2 Accuracy Test

For the cases used in the New Feature Test section, the features and the number of cases are recorded here. When multiple operations are tested, multiple tables are needed to include details of these operations.

Operation:

|Test Point           | Description                      | Quantity |  Comment |
|----------           |----------------------------------|----------|  --------|
|Data type test       |half/float/int8                   |          |          |
|Mult-tensor test     |Supports 1-8 dims                 |          |          |
|Layout test          |Supports NCHW/NHWC                |          |          |
|Zero element test    |Whether to support this test      |          |          |
|Stability test       |--gtest_repeat=NUM<br>--thread=NUM|          |          |
|Mult-platform test   |MLU370/MLU590                     |          |          |
|Nan/INF test         |Whether to support this test      |          |          |

### 3.3 Performance Test

See [MLU-OPS Performance Acceptance Standard](https://github.com/Cambricon/mlu-ops/blob/master/docs/MLU-OPS-Performance-Acceptance-Standard.md) for details.

Platform：MLU370

|Operation|Mlu_hardware_time(us)|Mlu_interface_time(us)|Mlu_io_efficiency|Mlu_compute_efficiency|Mlu_workwpace_size(Bytes)|Data_type|Shape|
|-------|----|----|-----|----|----|----|-----|
|op_name|    |    |     |    |    |    |     |
|op_name|    |    |     |    |    |    |     |

Platform：MLU590

|Operation|Mlu_hardware_time(us)|Mlu_interface_time(us)|Mlu_io_efficiency|Mlu_compute_efficiency|Mlu_workwpace_size(Bytes)|Data_type|Shape|
|-------|----|----|----|----|----|----|-----|
|op_name|    |    |    |    |    |    |     |
|op_name|    |    |    |    |    |    |     |

### 3.4 Summary Analysis

Please give a brief overview here, if you want to note and summarize the content.

### CODE REVIEW

**Review by NekHan — APPROVED** [review]
GOOD

File: bangc-ops/test/mlu_op_gtest/api_gtest/src/gtest/indice_convolution_forward/indice_convolution_forward_general.cpp [hunk 1]
```diff
@@ -0,0 +1,490 @@
...
+        inverse_, sub_m_, workspace_, workspace_size_, features_out_desc_,
+        features_out_);
+    destroy();
+    return expected_status_ == status;
+  }
+
+  void destroy() {
+    try {
+      if (handle_) {
+        // CNRT_CHECK(cnrtQueueSync(handle_->queue));
```
Comment (LittleReal) [inline bangc-ops/test/mlu_op_gtest/api_gtest/src/gtest/indice_convolution_forward/indice_convolution_forward_general.cpp]: 为什么sync注释掉了

**Review by LittleReal — APPROVED** [review]
LGTM

**Comment by Shin-Wang** [comment]
```
[ RUN      ] indice_convolution_forward.BAD_PARAM_handle_null
[2023-2-23 19:30:0] [MLUOP] [Error]:[mluOpIndiceConvolutionForward] Check failed: handle != nullptr. 
[       OK ] indice_convolution_forward.BAD_PARAM_handle_null (350 ms)
[ RUN      ] indice_convolution_forward.BAD_PARAM_features_desc_null
[2023-2-23 19:30:0] [MLUOP] [Error]:[mluOpIndiceConvolutionForward] Check failed: features_desc != nullptr. 
[       OK ] indice_convolution_forward.BAD_PARAM_features_desc_null (1 ms)
[ RUN      ] indice_convolution_forward.BAD_PARAM_features_null
[2023-2-23 19:30:0] [MLUOP] [Error]:[mluOpIndiceConvolutionForward] Check failed: features != nullptr. 
[       OK ] indice_convolution_forward.BAD_PARAM_features_null (0 ms)
[ RUN      ] indice_convolution_forward.BAD_PARAM_filters_desc_null
[2023-2-23 19:30:0] [MLUOP] [Error]:[mluOpIndiceConvolutionForward] Check failed: filters_desc != nullptr. 
[       OK ] indice_convolution_forward.BAD_PARAM_filters_desc_null (0 ms)
[ RUN      ] indice_convolution_forward.BAD_PARAM_filters_null
[2023-2-23 19:30:0] [MLUOP] [Error]:[mluOpIndiceConvolutionForward] Check failed: filters != nullptr. 
[       OK ] indice_convolution_forward.BAD_PARAM_filters_null (0 ms)
[ RUN      ] indice_convolution_forward.BAD_PARAM_indice_pairs_desc_null
[2023-2-23 19:30:0] [MLUOP] [Error]:[mluOpIndiceConvolutionForward] Check failed: indice_pairs_desc != nullptr. 
[       OK ] indice_convolution_forward.BAD_PARAM_indice_pairs_desc_null (0 ms)
[ RUN      ] indice_convolution_forward.BAD_PARAM_indice_pairs_null
[2023-2-23 19:30:0] [MLUOP] [Error]:[mluOpIndiceConvolutionForward] Check failed: indice_pairs != nullptr. 
[       OK ] indice_convolution_forward.BAD_PARAM_indice_pairs_null (0 ms)
[ RUN      ] indice_convolution_forward.BAD_PARAM_features_out_desc_null
[2023-2-23 19:30:0] [MLUOP] [Error]:[mluOpIndiceConvolutionForward] Check failed: features_out_desc != nullptr. 
[       OK ] indice_convolution_forward.BAD_PARAM_features_out_desc_null (0 ms)
[ RUN      ] indice_convolution_forward.BAD_PARAM_features_out_null
[2023-2-23 19:30:0] [MLUOP] [Error]:[mluOpIndiceConvolutionForward] Check failed: features_out != nullptr. 
[       OK ] indice_convolution_forward.BAD_PARAM_features_out_null (0 ms)
[ RUN      ] indice_convolution_forward.BAD_PARAM_indice_num_null
[2023-2-23 19:30:0] [MLUOP] [Error]:[mluOpIndiceConvolutionForward] Check failed: indice_num != nullptr. 
[       OK ] indice_convolution_forward.BAD_PARAM_indice_num_null (0 ms)
[ RUN      ] indice_convolution_forward.BAD_PARAM_workspace_null
[2023-2-23 19:30:0] [MLUOP] [Error]:[mluOpIndiceConvolutionForward] Check failed: workspace != nullptr. 
[       OK ] indice_convolution_forward.BAD_PARAM_workspace_null (0 ms)
[----------] 11 tests from indice_convolution_forward (351 ms total)

```

**Comment by Shin-Wang** [comment]
```
[----------] 7 tests from indice_convolution_forward_workspace
[ RUN      ] indice_convolution_forward_workspace.BAD_PARAM_handle_null
[2023-2-23 19:30:0] [MLUOP] [Error]:[mluOpGetIndiceConvolutionForwardWorkspaceSize] Check failed: handle != nullptr. 
[       OK ] indice_convolution_forward_workspace.BAD_PARAM_handle_null (0 ms)
[ RUN      ] indice_convolution_forward_workspace.BAD_PARAM_features_desc_null
[2023-2-23 19:30:0] [MLUOP] [Error]:[mluOpGetIndiceConvolutionForwardWorkspaceSize] Check failed: features_desc != nullptr. 
[       OK ] indice_convolution_forward_workspace.BAD_PARAM_features_desc_null (0 ms)
[ RUN      ] indice_convolution_forward_workspace.BAD_PARAM_filters_desc_null
[2023-2-23 19:30:0] [MLUOP] [Error]:[mluOpGetIndiceConvolutionForwardWorkspaceSize] Check failed: filters_desc != nullptr. 
[       OK ] indice_convolution_forward_workspace.BAD_PARAM_filters_desc_null (0 ms)
[ RUN      ] indice_convolution_forward_workspace.BAD_PARAM_indice_pairs_desc_null
[2023-2-23 19:30:0] [MLUOP] [Error]:[mluOpGetIndiceConvolutionForwardWorkspaceSize] Check failed: indice_pairs_desc != nullptr. 
[       OK ] indice_convolution_forward_workspace.BAD_PARAM_indice_pairs_desc_null (0 ms)
[ RUN      ] indice_convolution_forward_workspace.BAD_PARAM_features_out_desc_null
[2023-2-23 19:30:0] [MLUOP] [Error]:[mluOpGetIndiceConvolutionForwardWorkspaceSize] Check failed: features_out_desc != nullptr. 
[       OK ] indice_convolution_forward_workspace.BAD_PARAM_features_out_desc_null (0 ms)
[ RUN      ] indice_convolution_forward_workspace.BAD_PARAM_indice_num_null
[2023-2-23 19:30:0] [MLUOP] [Error]:[mluOpGetIndiceConvolutionForwardWorkspaceSize] Check failed: indice_num != nullptr. 
[       OK ] indice_convolution_forward_workspace.BAD_PARAM_indice_num_null (0 ms)
[ RUN      ] indice_convolution_forward_workspace.BAD_PARAM_workspace_size_null
[2023-2-23 19:30:0] [MLUOP] [Error]:[mluOpGetIndiceConvolutionForwardWorkspaceSize] Check failed: size != nullptr. 
[       OK ] indice_convolution_forward_workspace.BAD_PARAM_workspace_size_null (0 ms)
[----------] 7 tests from indice_convolution_forward_workspace (0 ms total)

```

**Comment by Shin-Wang** [comment]
```
[----------] 1 test from zero_element_1/indice_convolution_forward_general
[ RUN      ] zero_element_1/indice_convolution_forward_general.negative/0
[       OK ] zero_element_1/indice_convolution_forward_general.negative/0 (0 ms)
[----------] 1 test from zero_element_1/indice_convolution_forward_general (0 ms total)

[----------] 1 test from zero_element_2/indice_convolution_forward_general
[ RUN      ] zero_element_2/indice_convolution_forward_general.negative/0
[       OK ] zero_element_2/indice_convolution_forward_general.negative/0 (0 ms)
[----------] 1 test from zero_element_2/indice_convolution_forward_general (0 ms total)

[----------] 4 tests from bad_features_dtype_dim_shape/indice_convolution_forward_general
[ RUN      ] bad_features_dtype_dim_shape/indice_convolution_forward_general.negative/0
[2023-2-23 19:30:0] [MLUOP] [Error]:[mluOpGetIndiceConvolutionForwardWorkspaceSize] Check failed: features_desc->dtype == features_out_desc->dtype && features_desc->dtype == filters_desc->dtype. 
[       OK ] bad_features_dtype_dim_shape/indice_convolution_forward_general.negative/0 (0 ms)
[ RUN      ] bad_features_dtype_dim_shape/indice_convolution_forward_general.negative/1
[2023-2-23 19:30:0] [MLUOP] [Error]:[mluOpGetIndiceConvolutionForwardWorkspaceSize] Check failed: features_desc->dim == 2. 
[       OK ] bad_features_dtype_dim_shape/indice_convolution_forward_general.negative/1 (0 ms)
[ RUN      ] bad_features_dtype_dim_shape/indice_convolution_forward_general.negative/2
[2023-2-23 19:30:0] [MLUOP] [Error]:[mluOpGetIndiceConvolutionForwardWorkspaceSize] Check failed: features_desc->dims[1] == ci. 
[       OK ] bad_features_dtype_dim_shape/indice_convolution_forward_general.negative/2 (0 ms)
[ RUN      ] bad_features_dtype_dim_shape/indice_convolution_forward_general.negative/3
[2023-2-23 19:30:0] [MLUOP] [Error]:[mluOpGetIndiceConvolutionForwardWorkspaceSize] Check failed: features_desc->dims[0] == indice_pairs_desc->dims[2]. 
[       OK ] bad_features_dtype_dim_shape/indice_convolution_forward_general.negative/3 (0 ms)
[----------] 4 tests from bad_features_dtype_dim_shape/indice_convolution_forward_general (0 ms total)

[----------] 1 test from bad_filters_dtype/indice_convolution_forward_general
[ RUN      ] bad_filters_dtype/indice_convolution_forward_general.negative/0
[2023-2-23 19:30:0] [MLUOP] [Error]:[mluOpGetIndiceConvolutionForwardWorkspaceSize] Check failed: features_desc->dtype == features_out_desc->dtype && features_desc->dtype == filters_desc->dtype. 
[       OK ] bad_filters_dtype/indice_convolution_forward_general.negative/0 (0 ms)
[----------] 1 test from bad_filters_dtype/indice_convolution_forward_general (0 ms total)

[----------] 2 tests from bad_filters_dim_layout/indice_convolution_forward_general
[ RUN      ] bad_filters_dim_layout/indice_convolution_forward_general.negative/0
[2023-2-23 19:30:0] [MLUOP] [Error]:[mluOpGetIndiceConvolutionForwardWorkspaceSize]The filters dimension number only support 5 currently, but filters dimension number is :4.
[       OK ] bad_filters_dim_layout/indice_convolution_forward_general.negative/0 (0 ms)
[ RUN      ] bad_filters_dim_layout/indice_convolution_forward_general.negative/1
[2023-2-23 19:30:0] [MLUOP] [Error]:[mluOpGetIndiceConvolutionForwardWorkspaceSize]The layout of filters is: LAYOUT_NHWC, which is not supported now.
[       OK ] bad_filters_dim_layout/indice_convolution_forward_general.negative/1 (1 ms)
[----------] 2 tests from bad_filters_dim_layout/indice_convolution_forward_general (1 ms total)

[----------] 5 tests from bad_indice_pairs_dtype_dim_shape/indice_convolution_forward_general
[ RUN      ] bad_indice_pairs_dtype_dim_shape/indice_convolution_forward_general.negative/0
[2023-2-23 19:30:0] [MLUOP] [Error]:[mluOpGetIndiceConvolutionForwardWorkspaceSize] Check failed: indice_pairs_desc->dtype == MLUOP_DTYPE_INT32. 
[       OK ] bad_indice_pairs_dtype_dim_shape/indice_convolution_forward_general.negative/0 (0 ms)
[ RUN      ] bad_indice_pairs_dtype_dim_shape/indice_convolution_forward_general.negative/1
[2023-2-23 19:30:0] [MLUOP] [Error]:[mluOpGetIndiceConvolutionForwardWorkspaceSize] Check failed: indice_pairs_desc->dim == 3. 
[       OK ] bad_indice_pairs_dtype_dim_shape/indice_convolution_forward_general.negative/1 (0 ms)
[ RUN      ] bad_indice_pairs_dtype_dim_shape/indice_convolution_forward_general.negative/2
[2023-2-23 19:30:0] [MLUOP] [Error]:[mluOpGetIndiceConvolutionForwardWorkspaceSize] Check failed: indice_pairs_desc->dims[0] == num_filter. 
[       OK ] bad_indice_pairs_dtype_dim_shape/indice_convolution_forward_general.negative/2 (0 ms)
[ RUN      ] bad_indice_pairs_dtype_dim_shape/indice_convolution_forward_general.negative/3
[2023-2-23 19:30:0] [MLUOP] [Error]:[mluOpGetIndiceConvolutionForwardWorkspaceSize] Check failed: indice_pairs_desc->dims[1] == 2. 
[       OK ] bad_indice_pairs_dtype_dim_shape/indice_convolution_forward_general.negative/3 (0 ms)
[ RUN      ] bad_indice_pairs_dtype_dim_shape/indice_convolution_forward_general.negative/4
[2023-2-23 19:30:0] [MLUOP] [Error]:[mluOpGetIndiceConvolutionForwardWorkspaceSize] Check failed: features_desc->dims[0] == indice_pairs_desc->dims[2]. 
[       OK ] bad_indice_pairs_dtype_dim_shape/indice_convolution_forward_general.negative/4 (0 ms)
[----------] 5 tests from bad_indice_pairs_dtype_dim_shape/indice_convolution_forward_general (0 ms total)

[----------] 4 tests from bad_features_output_dtype_dim_shape/indice_convolution_forward_general
[ RUN      ] bad_features_output_dtype_dim_shape/indice_convolution_forward_general.negative/0
[2023-2-23 19:30:0] [MLUOP] [Error]:[mluOpGetIndiceConvolutionForwardWorkspaceSize] Check failed: features_desc->dtype == features_out_desc->dtype && features_desc->dtype == filters_desc->dtype. 
[       OK ] bad_features_output_dtype_dim_shape/indice_convolution_forward_general.negative/0 (0 ms)
[ RUN      ] bad_features_output_dtype_dim_shape/indice_convolution_forward_general.negative/1
[2023-2-23 19:30:0] [MLUOP] [Error]:[mluOpGetIndiceConvolutionForwardWorkspaceSize] Check failed: features_out_desc->dim == 2. 
[       OK ] bad_features_output_dtype_dim_shape/indice_convolution_forward_general.negative/1 (0 ms)
[ RUN      ] bad_features_output_dtype_dim_shape/indice_convolution_forward_general.negative/2
[2023-2-23 19:30:0] [MLUOP] [Error]:[mluOpGetIndiceConvolutionForwardWorkspaceSize] Check failed: features_out_desc->dims[0] == num_act_out. 
[       OK ] bad_features_output_dtype_dim_shape/indice_convolution_forward_general.negative/2 (0 ms)
[ RUN      ] bad_features_output_dtype_dim_shape/indice_convolution_forward_general.negative/3
[2023-2-23 19:30:0] [MLUOP] [Error]:[mluOpGetIndiceConvolutionForwardWorkspaceSize] Check failed: features_out_desc->dims[1] == co. 
[       OK ] bad_features_output_dtype_dim_shape/indice_convolution_forward_general.negative/3 (0 ms)
[----------] 4 tests from bad_features_output_dtype_dim_shape/indice_convolution_forward_general (0 ms total)

[----------] 5 tests from bad_params/indice_convolution_forward_general
[ RUN      ] bad_params/indice_convolution_forward_general.negative/0
[2023-2-23 19:30:0] [MLUOP] [Error]:[mluOpGetIndiceConvolutionForwardWorkspaceSize] Check failed: indice_num[i] >= 0 && indice_num[i] <= features_desc->dims[0]. 
[       OK ] bad_params/indice_convolution_forward_general.negative/0 (0 ms)
[ RUN      ] bad_params/indice_convolution_forward_general.negative/1
[2023-2-23 19:30:0] [MLUOP] [Error]:[mluOpGetIndiceConvolutionForwardWorkspaceSize] Check failed: indice_num[i] >= 0 && indice_num[i] <= features_desc->dims[0]. 
[       OK ] bad_params/indice_convolution_forward_general.negative/1 (0 ms)
[ RUN      ] bad_params/indice_convolution_forward_general.negative/2
[2023-2-23 19:30:0] [MLUOP] [Error]:[mluOpGetIndiceConvolutionForwardWorkspaceSize] Check failed: features_out_desc->dims[0] == num_act_out. 
[       OK ] bad_params/indice_convolution_forward_general.negative/2 (0 ms)
[ RUN      ] bad_params/indice_convolution_forward_general.negative/3
[2023-2-23 19:30:0] [MLUOP] [Error]:[mluOpGetIndiceConvolutionForwardWorkspaceSize] Check failed: inverse == 0 || inverse == 1. 
[       OK ] bad_params/indice_convolution_forward_general.negative/3 (0 ms)
[ RUN      ] bad_params/indice_convolution_forward_general.negative/4
[2023-2-23 19:30:0] [MLUOP] [Error]:[mluOpGetIndiceConvolutionForwardWorkspaceSize] Check failed: sub_m == 0 || sub_m == 1. 
[       OK ] bad_params/indice_convolution_forward_general.negative/4 (0 ms)
[----------] 5 tests from bad_params/indice_convolution_forward_general (0 ms total)

[----------] 1 test from bad_params_inverse/indice_convolution_forward_general
[ RUN      ] bad_params_inverse/indice_convolution_forward_general.negative/0
[2023-2-23 19:30:0] [MLUOP] [Error]:[mluOpGetIndiceConvolutionForwardWorkspaceSize]inverse is: 1, which is not supported now.
[       OK ] bad_params_inverse/indice_convolution_forward_general.negative/0 (0 ms)
[----------] 1 test from bad_params_inverse/indice_convolution_forward_general (0 ms total)

[----------] 1 test from bad_large_tensor_features/indice_convolution_forward_general
[ RUN      ] bad_large_tensor_features/indice_convolution_forward_general.negative/0
[2023-2-23 19:30:0] [MLUOP] [Warning]:[mluOpSetTensorDescriptor]: overflow max tensor num.Currently, mluOp supports tensor num smaller than 2^31,now tensor dims:(21474837,100), data_width:4.
[2023-2-23 19:30:0] [MLUOP] [Error]:[mluOpGetIndiceConvolutionForwardWorkspaceSize]Max tensor number overflow. Currently, MLU-OPS supports tensor elemenets number smaller than 2^31.
[       OK ] bad_large_tensor_features/indice_convolution_forward_general.negative/0 (1 ms)
[----------] 1 test from bad_large_tensor_features/indice_convolution_forward_general (1 ms total)

[----------] 1 test from bad_large_tensor_filters/indice_convolution_forward_general
[ RUN      ] bad_large_tensor_filters/indice_convolution_forward_general.negative/0
[2023-2-23 19:30:0] [MLUOP] [Warning]:[mluOpSetTensorDescriptor]: overflow max tensor num.Currently, mluOp supports tensor num smaller than 2^31,now tensor dims:(89479,2,2,2,3000), data_width:4.
[2023-2-23 19:30:0] [MLUOP] [Error]:[mluOpGetIndiceConvolutionForwardWorkspaceSize]Max tensor number overflow. Currently, MLU-OPS supports tensor elemenets number smaller than 2^31.
[       OK ] bad_large_tensor_filters/indice_convolution_forward_general.negative/0 (0 ms)
[----------] 1 test from bad_large_tensor_filters/indice_convolution_forward_general (0 ms total)

[----------] 1 test from bad_large_tensor_indice_pairs/indice_convolution_forward_general
[ RUN      ] bad_large_tensor_indice_pairs/indice_convolution_forward_general.negative/0
[2023-2-23 19:30:0] [MLUOP] [Warning]:[mluOpSetTensorDescriptor]: overflow max tensor num.Currently, mluOp supports tensor num smaller than 2^31,now tensor dims:(12,2,89478486), data_width:4.
[2023-2-23 19:30:0] [MLUOP] [Error]:[mluOpGetIndiceConvolutionForwardWorkspaceSize]Max tensor number overflow. Currently, MLU-OPS supports tensor elemenets number smaller than 2^31.
[       OK ] bad_large_tensor_indice_pairs/indice_convolution_forward_general.negative/0 (2 ms)
[----------] 1 test from bad_large_tensor_indice_pairs/indice_convolution_forward_general (2 ms total)

[----------] 1 test from bad_large_tensor_features_output/indice_convolution_forward_general
[ RUN      ] bad_large_tensor_features_output/indice_convolution_forward_general.negative/0
[2023-2-23 19:30:0] [MLUOP] [Warning]:[mluOpSetTensorDescriptor]: overflow max tensor num.Currently, mluOp supports tensor num smaller than 2^31,now tensor dims:(4294968,500), data_width:4.
[2023-2-23 19:30:0] [MLUOP] [Error]:[mluOpGetIndiceConvolutionForwardWorkspaceSize]Max tensor number overflow. Currently, MLU-OPS supports tensor elemenets number smaller than 2^31.
[       OK ] bad_large_tensor_features_output/indice_convolution_forward_general.negative/0 (0 ms)
[----------] 1 test from bad_large_tensor_features_output/indice_convolution_forward_general (0 ms total)

```

**Review by GuangpingYuan — APPROVED** [review]
LGTM

**Comment by Shin-Wang** [comment]
测试报告：添加indice_convolution_forward防呆检查

**Pull request merged**